site/docs/v1/tech/apis/system.adoc

```diff
@@ -335,23 +335,62 @@ The specific contents of the JSON body are not documented here. If you choose to
 [since]#Available since 1.19.0#
 
 |453
-|The FusionAuth database connection pool connectivity is below the healthy threshold. Additional information may be available in the JSON response which be retrieved when using an API key.
+|The FusionAuth database connection pool connectivity is below the healthy threshold. Additional information may be available in the JSON response which is retrieved when using an API key.
 
 [since]#Available since 1.19.0#
 
 |454
-|The FusionAuth database connectivity pool connectivity is below the healthy threshold. Additional information may be available in the JSON response which be retrieved when using an API key.
+|The FusionAuth database connectivity pool connectivity is below the healthy threshold. Additional information may be available in the JSON response which is retrieved when using an API key.
 
 [since]#Available since 1.19.0#
 
 |460
-|FusionAuth is using Elasticsearch and the the search service is reporting an unhealthy cluster status. In a cluster with 2+ nodes, this mean the cluster status is being reported as `yellow` or `red`. In a single node Elasticsearch configuration this means the cluster status is `red.`
+|FusionAuth is using Elasticsearch and the search service is reporting an unhealthy cluster status. In a cluster with 2+ nodes, this means the cluster status is being reported as `yellow` or `red`. In a single-node Elasticsearch configuration this means the cluster status is `red.[.line-through]#`#
```

Comment:
was this a spacing issue? the change to the period after red?

Reply:
Not sure what happened there - must have been between the copy and the paste of Grammarly.  Reverting to before (`red`).